File: PHP/Maskify/MySolution.php
<?php

function maskify(string $cc): string {
	return strlen($cc) <= 4 ? $cc : str_repeat("#", (strlen($cc)-4)) . substr($cc, (strlen($cc)-4));
}

// Original kata: https://www.codewars.com/kata/5412509bd436bd33920011bc
// My solution: https://www.codewars.com/kata/reviews/672ed9269c0da96bae9feb5f/groups/675aaaa922b5eacdfd7c5bcf

File: CPP/SmallOnArray/MySolution.cpp
using namespace std; 

int findSmallest(vector <int> list)
{
  return *min_element(list.begin(), list.end());
}

// Original kata: https://www.codewars.com/kata/55a2d7ebe362935a210000b2
// My solution: https://www.codewars.com/kata/reviews/5ae2f7e3fa02e0b5f9000af5/groups/634883fe73b43e00011dfe2c
